spec/controllers/carts_controller_spec.rb

```diff
@@ -1,52 +1,17 @@
 describe CartsController do
-  let(:user1) { FactoryGirl.create(:user) }
-  let(:user2) { FactoryGirl.create(:user) }
-  let(:user3) { FactoryGirl.create(:user) }
-  let(:user4) { FactoryGirl.create(:user) }
+  let(:user) { FactoryGirl.create(:user) }
   let(:approval_group1) { FactoryGirl.create(:approval_group, name: 'test-approval-group1') }
-  let(:approval_group2) { FactoryGirl.create(:approval_group, name: 'test-approval-group2') }
 
   before do
-    UserRole.create!(user_id: user1.id, approval_group_id: approval_group1.id, role: 'approver')
-    UserRole.create!(user_id: user2.id, approval_group_id: approval_group1.id, role: 'requester')
-    UserRole.create!(user_id: user3.id, approval_group_id: approval_group1.id, role: 'approver')
+    UserRole.create!(user_id: user.id, approval_group_id: approval_group1.id, role: 'requester')
     p = {'approvalGroup' => 'test-approval-group1', 'cartName' => 'cart1' }
     @cart1 = Commands::Approval::InitiateCartApproval.new.perform(p)
     session[:user] = {}
   end
 
-  describe('index') do
-
-    it 'should find the open cart' do
-      session[:user]['email'] = user2.email_address
-      get :index
-      expect(assigns(:open_carts).first).to eq(@cart1)
-    end
-
-    it 'should find nothing' do
-      session[:user]['email'] = user4.email_address
-      get :index
-      expect(assigns(:open_carts).first).to be_nil
-    end
-
-    it 'should find the approved cart' do
-      session[:user]['email'] = user2.email_address
-      @cart1.update_attributes(status: 'approved')
-      get :index
-      expect(assigns(:closed_carts).first).to eq(@cart1)
-    end
-
-    it 'should find the rejected cart' do
-      session[:user]['email'] = user2.email_address
-      @cart1.update_attributes(status: 'rejected')
-      get :index
-      expect(assigns(:closed_carts).first).to eq(@cart1)
-    end
-  end
```

Comment:
Probably worth leaving a basic test, right? Or are there other tests that cover it?